pvanalytics/quality/gaps.py

```diff
@@ -409,8 +409,11 @@ def trim(series, days=10):
     """
     start, end = start_stop_dates(series, days=days)
     mask = pd.Series(False, index=series.index)
+
     if start:
-        mask.loc[start.date():end.date()] = True
+        mask.loc[pd.to_datetime(start.date()).tz_localize(series.index.tz):
+                 pd.to_datetime(end.date()).tz_localize(series.index.tz)] = \
+            True
```

Comment:
The return values of `start_stop_dates` seem to always have timestamps at midnight.  With that in mind, do we need to do any conversion like `.date()` here?  Couldn't we just use `start` and `end` as-is?

Reply:
You are right mask.loc[start:end] = True would also works since adding .date() would change start and end from tz-aware to tz-naive, which made the Pandas error. I edited the code and have pushed your suggestion.